frontend/src/component/admin/apiToken/ApiTokenTable/ProjectApiTokenCreate.tsx

```diff
@@ -1,29 +0,0 @@
-import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
-import useProjectAccess from 'hooks/api/getters/useProjectAccess/useProjectAccess';
-import { useAccess } from 'hooks/api/getters/useAccess/useAccess';
-import { GO_BACK } from 'constants/navigate';
-import { CreateApiToken } from '../CreateApiToken/CreateApiToken';
-import { SidebarModal } from 'component/common/SidebarModal/SidebarModal';
-import { useNavigate } from 'react-router-dom';
-
-export const ProjectApiTokenCreate = () => {
-    const projectId = useRequiredPathParam('projectId');
-    const navigate = useNavigate();
-
-    const { access } = useProjectAccess(projectId);
-    const { users, serviceAccounts, groups } = useAccess();
-
-    if (!access || !users || !serviceAccounts || !groups) {
-        return null;
-    }
```

Comment:
Removed this from the new component because it didn't make sense to me. Why do we need to check access, users, service accounts or groups to create an API token here?

Reply:
Thanks, I think that was left there by mistake